visualization/app/codeCharta/ui/mapTreeView/mapTreeViewLevelComponent.spec.ts

```diff
@@ -197,17 +210,22 @@ describe("MapTreeViewLevelController", () => {
         });
 
         it("Subscribe Hover", () => {
-            let subscriber: MapTreeViewHoverEventSubscriber = {
-                onShouldHoverNode: jest.fn(),
-                onShouldUnhoverNode: jest.fn()
-            };
+
+
+            mapTreeViewLevelController.node = codeMapUtilService.getCodeMapNodeFromPath("/root/a/ab/aba", "File");
 
             MapTreeViewLevelController.subscribeToHoverEvents($rootScope, subscriber);
 
             expect($rootScope.$on).toBeCalledWith("should-hover-node", expect.any(Function));
             expect($rootScope.$on).toBeCalledWith("should-unhover-node", expect.any(Function));
-            expect(subscriber.onShouldHoverNode).toBeCalled;
-            expect(subscriber.onShouldUnhoverNode).toBeCalled;
+
+            $rootScope.$on("should-hover-node", (event: IAngularEvent, node: CodeMapNode) => {
+                expect(node).toBe(mapTreeViewLevelController.node);
+            });
+
+            $rootScope.$on("should-unhover-node", (event: IAngularEvent, node: CodeMapNode) => {
+                expect(node).toBe(mapTreeViewLevelController.node);
```

Comment:
Unfortunately the test still passes with random values like:
`expect(node).toBe("whatever");`